examples/game_of_life/_helpers.py

```diff
@@ -26,9 +26,13 @@ def get_active_pin_map_id(self) -> str:
         Returns:
             The resource id of the pin map that is registered to the pin map service.
         """
-        return self._sequence_context.Execution.RunTimeVariables.GetValString(
-            "NI.MeasurementLink.PinMapId", 0x0
-        )
+        PIN_MAP_ID = "NI.MeasurementLink.PinMapId"
```

Comment:
I think this should be a constant at the top level of the file and internal like:
```
_PIN_MAP_ID = "NI.MeasurementLink.PinMapId"
```

Reply:
@bkeryan ?

Reply:
I agree. Also rename to indicate that this is a runtime var name, not the actual id

Reply:
Modified.